tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowsersecuritychange_event.html

```diff
@@ -0,0 +1,46 @@
+<head>
+  <title>mozbrowsersecuritychange</title>
```

Comment:
How about "mozbrowsersecuritychange event is dispatched when content security state changes"